packages/amplify_datastore/ios/Classes/CognitoPlugin.swift

```diff
@@ -22,259 +22,140 @@ public class CognitoPlugin: AuthCategoryPlugin {
         self.nativeAuthPlugin = nativeAuthPlugin
     }
     
-
-    public func signIn(
-        withUrlUrl url: String,
-        callbackUrlScheme: String,
-        preferPrivateSession: NSNumber,
-        browserPackageName: String?
-    ) async -> ([String : String]?, FlutterError?) {
-        preconditionFailure("signIn is not supported")
-    }
-    
-    public func signOut(
-        withUrlUrl url: String,
-        callbackUrlScheme: String,
-        preferPrivateSession: NSNumber,
-        browserPackageName: String?
-    ) async -> FlutterError? {
-        preconditionFailure("signOut is not supported")
-    }
-    
     public let key: PluginKey = "awsCognitoAuthPlugin"
     
     public func configure(using configuration: Any?) throws {}
     
-    public func reset(onComplete: @escaping BasicClosure) {
-        onComplete()
+// TODO: Migrate to Async Swift v2
+//    public func reset(onComplete: @escaping BasicClosure) {
+//        onComplete()
+//    }
```

Comment:
*nit:* Could this function throw an exception until it is migrated rather than committing commented out code?

Reply:
Looks like this is only used in testing, but I added a stub for it.